packages/carousel/src/react/index.js

```diff
@@ -185,36 +183,37 @@ const Pages = React.forwardRef((props, ref) => {
       id={context.id}
       ref={ref}
       role="region"
-      tabIndex="0"
       {...styles.pages(props)}
       {...filterReactProps(props)}
     />
   )
 })
 
+Pages.displayName = 'Carousel.Pages'
+
 Pages.propTypes = {
   onSwipeLeft: PropTypes.func.isRequired,
   onSwipeRight: PropTypes.func.isRequired
 }
 
-function Page(props) {
+const Page = props => {
   const ref = React.useRef()
 
   const { children, isActivePage, ...rest } = props
   const { offset } = React.useContext(CarouselContext)
-
   return (
     <li
       ref={ref}
       {...styles.page(props)}
       {...css({ transform: `translate3d(${offset}px, 0, 0)` })}
       {...rest}
-      {...(!isActivePage && { hidden: true, tabIndex: -1 })}
+      {...(!isActivePage && { hidden: true })}
```

Comment:
These must have been tabIndex -1 for a reason (programmatic focus).  Where was that happening?  I don't see it removed in this diff.

Reply:
**li** by default have no tabinddex. The purpose of `tabIndex: -1` is to explicitly make sure an element is not reachable via sequential keyboard navigation, but could be focused with Javascript or visually.  This particular **li** is the Page or visible section of the **Carousel**.  One of the objectives was remove tab focus from this element. This is just a semantically clean implementation.

Reply:
Yeah, the fact that tabIndex=-1 is used for programmatic focus was the point of my question.  Where is the code that was programmatically focusing?  I don't see it removed.  Maybe it should be?